src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -33,6 +33,30 @@
         <property name="username" value="ca"/>
         <property name="password" value=""/>
     </bean>
+
+
+    <bean id="flightDAO" class="com.epam.lowcost.DAO.FlightDAOImpl">
+        <constructor-arg ref="idDataSource"/>
+    </bean>
+
+
+    <bean id="flightService" class="com.epam.lowcost.service.FlightServiceImpl">
+        <constructor-arg ref="flightDAO"/>
+        <constructor-arg ref="planeService"/>
+    </bean>
+
+
+    <bean
+            class="org.springframework.web.servlet.view.InternalResourceViewResolver">
+        <property name="prefix" value="/views/" />
+        <property name="suffix" value=".jsp" />
+    </bean>
+
+    <bean id="idDataSource" class="org.h2.jdbcx.JdbcDataSource">
```

Comment:
This bean is no longer needed. Remove it, please

Reply:
done